### PR TITLE
Prevent not needed `<hr>` in more-info-light

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

If you have a light that supports brightness and color temperature, the more-info dialog added a `<hr>` below the color temperature slider, although there is no further UI section below it.
image

With this PR the `<hr>` only gets added if there is actually something below the color temperature such as an effect selection, by given the "authority" to add that `<hr>` to the effect rendering section (the color temperature | color selection area already had the "authority" to add its own `<hr>`).

Confirmed in gallery that everything looks good.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-light.ts [hunk 1]
```diff
@@ -147,72 +146,66 @@ class MoreInfoLight extends LitElement {
                         class="segmentationButton"
                       ></ha-icon-button>
                     </div>
-                    
-                    ${
-                      supportsRgbw || supportsRgbww
-                        ? html`<ha-labeled-slider
+
```
Comment (spacegaier) [inline src/dialogs/more-info/controls/more-info-light.ts:149]: No idea why GitHub marks that many changes. I only changed/moved lines that contain `<hr></hr>` (two removals, one addition).

**Comment by bramkragten** [comment]
It could have attributes still...

**Comment by spacegaier** [comment]
True, but perhaps the `<ha-attributes>` should add a `<hr>` itself in general? Currently there is none, example sensor:

image

**Comment by bramkragten** [comment]
> True, but perhaps the `<ha-attributes>` should add a `<hr>` itself in general? Currently there is none, example sensor:
> 

I think that could look nice.

**Review by bramkragten — APPROVED** [review]

File: src/dialogs/more-info/controls/more-info-light.ts (hunk 1 above)
Comment (kukulich) [reply src/dialogs/more-info/controls/more-info-light.ts:149]: You probably modify whitespace - see without whitespace changes: https://github.com/home-assistant/frontend/pull/9080/files?w=1

**Pull request merged**